src/main/webapp/app/localci/build-agents/build-agent-summary/build-agent-summary.component.html

```diff
@@ -75,7 +83,7 @@ <h3 id="build-agents-heading" jhiTranslate="artemisApp.buildAgents.summary"></h3
                         <ng-template ngx-datatable-cell-template let-value="value" let-row="row">
                             <span style="margin-right: 20px">{{ value }}</span>
                             @if (value > 0) {
-                                <button class="btn btn-danger btn-sm" (click)="cancelAllBuildJobs(row.name)">
+                                <button class="btn btn-danger btn-sm" (click)="cancelAllBuildJobs(row.buildAgent!.name)">
```

Comment:
How about using `row.buildAgend` (might be `undefined`) as an argument here? We could then do the null-check in the `cancelAllBuildJobs` function and gather the name if it's existent?